modules/tinymce/src/plugins/link/test/ts/browser/UrlInputTest.ts

```diff
@@ -15,6 +16,17 @@ describe('browser.tinymce.plugins.link.UrlInputTest', () => {
     base_url: '/project/tinymce/js/tinymce'
   }, [ Plugin ]);
 
+  const platform = PlatformDetection.detect();
```

Comment:
Please place this above the hook definition.

Reply:
https://github.com/tinymce/tinymce/pull/7533/commits/cadb868284e28a3e86bcf7107b4cd1a2adc70f78